graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/tweetfeature/TweetFeatureTest.java

```diff
@@ -0,0 +1,37 @@
+package com.twitter.graphjet.algorithms.counting.tweetfeature;
+
+import com.twitter.graphjet.algorithms.counting.tweetfeature.TweetFeature;
+import org.junit.Assert;
+import org.junit.Rule;
```

Comment:
I believe junit.Rule for this class isn't necessary as we're not using timeout or expectedException